master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Felix Configuration Resource (FelixConfiguration)
+---
+
+A Felix configuration resource (FelixConfiguration) represents Felix configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `felixconfiguration`, `felixconfig`, `felixconfigurations`, `felixconfigs`.
+
+See [Configuring Felix]({{site.baseurl}}/{{page.version}}/referance/felix/configuration) for more details.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  ipv6Support: false
+  ignoreLooseRPF: true
+  chainInsertMode: Append
+```
+
+### Felix Configuration Definition
+
+#### Metadata
+
+| Field       | Description                 | Accepted Values   | Schema |
+|-------------|-----------------------------|-------------------|--------|
+| name     | Unique name to describe this resource instance. Required. | Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
+
+- Resource with the name "default" has a specific meaning - this contains the Felix global default configuration.
```

Comment:
Thanks again for explaining this one to me. I would suggest changing this sentence then to something more like, "Calico automatically creates a resource named `default` containing the global default configuration settings for Felix. You can use `calicoctl` to view and edit these settings."